octoprint_mrbeam/__init__.py

```diff
@@ -104,6 +106,7 @@
 from octoprint_mrbeam.util.uptime import get_uptime, get_uptime_human_readable
 from octoprint_mrbeam.util import get_thread
 from octoprint_mrbeam import camera
+from .version_comparator import VersionComparator
```

Comment:
please change it to octoprint_mrbeam.version.comperator

Reply:
done